File: routeMap/app/src/main/java/ru/android/zheka/gmapexample1/edit/Editable.java
package ru.android.zheka.gmapexample1.edit;

import com.activeandroid.Model;

import java.lang.reflect.Field;

public class Editable{
	private String name;
	public Model model;
	public Editable(Model model){
		this.model = model;
		try{name = (String) getField().get(model);
		}catch(IllegalAccessException e){
			e.printStackTrace();
		}catch (IllegalArgumentException e) {
			e.printStackTrace();
		}
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		try{getField().set(model, name);
		}catch(IllegalAccessException e){
			e.printStackTrace();
		}catch (IllegalArgumentException e) {
			e.printStackTrace();
		}
		this.name = name;
	}
	private Field getField(){
		Field field=null;
		try{field = model.getClass().getField("name");
		}catch(NoSuchFieldException e){
			e.printStackTrace();
		}catch (SecurityException e) {
			e.printStackTrace();
		}
		return field;
	}
}

File: routeMap/app/src/main/java/ru/android/zheka/gmapexample1/edit/Editable.kt
package ru.android.zheka.gmapexample1.edit

import com.activeandroid.Model
import java.lang.reflect.Field

class Editable(var model: Model) {
    var name: String? = null
//    fun getName(): String? {
//        return name
//    }

//    fun setName(name: String?) {
//        try {
//            field!![model] = name
//        } catch (e: IllegalAccessException) {
//            e.printStackTrace()
//        } catch (e: IllegalArgumentException) {
//            e.printStackTrace()
//        }
//        this.name = name
//    }

    private val field: Field?
        private get() {
            var field: Field? = null
            try {
                field = model.javaClass.getField("name")
            } catch (e: NoSuchFieldException) {
                e.printStackTrace()
            } catch (e: SecurityException) {
                e.printStackTrace()
            }
            return field
        }

    init {
        try {
            name = field!![model] as String
        } catch (e: IllegalAccessException) {
            e.printStackTrace()
        } catch (e: IllegalArgumentException) {
            e.printStackTrace()
        }
    }
}